### PR TITLE
update favourite layout

### PR DESCRIPTION
**Pull request by daliyajohn**
Update your favorites section UI

### CODE REVIEW

**Comment by tomahawk-pilot** [comment]
@daliyajohn can you share screenshot

File: src/components/Favourites/FavouriteCard.js [hunk 1]
```diff
@@ -61,63 +61,60 @@ export default function FavouriteCard(props) {
...
-                                        <Icon name="date" size={4} />
-                                    </div>
-                                    <p className="text-l">{props.day}</p>
-                                </div>
-                            </li> */}
-                        </ul>
+            <li className="py-1">
+                <div className="flex item-center space-x-1">
+                    <div className="flex items-center">
+                        <Icon name="date" size={4} />
```
Comment (athul) [inline src/components/Favourites/FavouriteCard.js:112]: Is this intentionally commented?

File: src/components/Favourites/FavouriteCard.js [hunk 2]
```diff
@@ -61,63 +61,60 @@ export default function FavouriteCard(props) {
...
                     </div>
+                    <p className="text-l">{props.day}</p>
                 </div>
+            </li> */}
```
Comment (athul) [inline src/components/Favourites/FavouriteCard.js:116]: Is this intentionally commented? Should this be deleted?

**Comment by daliyajohn** [comment]
> @daliyajohn can you share screenshot

**Review by athul — APPROVED** [review]

**Review by daliyajohn — COMMENTED** [review]
I have just changed the layout. That commented code is not mine. The initial stage itself that code is there. I just did some code indentation.

**Comment by athul** [comment]
Oh okay. Sorry 😅

**Pull request merged**